README.md

```diff
@@ -147,6 +147,7 @@ The below repos are direct forks of ps-mdt and have been edited to fit certain c
 
 # Credits
 * Originally Echo RP MDT released by [Flawws#999](https://github.com/FlawwsX/erp_mdt)
+* Mugshot Integration (https://github.com/CQC-Scripting/cqc-mugshot)

```

Comment:
Please change to 
* [CQC-Scripting](https://github.com/CQC-Scripting) for their [cqc-mugshot](https://github.com/CQC-Scripting/cqc-mugshot)

Reply:
done <3